### PR TITLE
Minor pylint fixes for tests

### PR DESCRIPTION
**Pull request by asankov**
The `too-many-ancestors` warning was displayed many times for different test classes - there we have a lot inheritance. The maximum `pylint` allows is 8. We can override it with a `.pylintrc` file, but since all of them were in tests, I suggest we disable it inline. This way it would still be valid for the business logic, where it may come handy.

Another minor fix - disabled `no-member` warning for `test_run.cc.count` - `test_run` is created through calling `TestRunFactory()`. `TestRunFactory` has a method `cc` and `TestRun` has a member `cc` (which has `.count()`). However, `pylint` thinks `test_run` is a `TestRunFactory` instance and `test_run.cc` is a reference to the method, which has no `.count()`. Clearly, `pylint` is wrong and that is why we silence it.

### CODE REVIEW

File: tcms/testruns/tests/test_views.py [hunk 1]
```diff
@@ -306,7 +306,7 @@ def test_404_if_run_not_exist(self):
         self.assert404(response)
 
     def assert_cc(self, response, expected_cc):
-        self.assertEqual(len(expected_cc), self.test_run.cc.count())
+        self.assertEqual(len(expected_cc), self.test_run.cc.count())  # pylint: disable=no-member
```
Comment (atodorov) [inline tcms/testruns/tests/test_views.py]: Revert this change. That sounds like an issue with pylint-django and it should be fixed there. The rest is fine.

Comment (asankov) [reply tcms/testruns/tests/test_views.py]: Done. IMO it is just not able to infer the type of `test_run` correctly
